### ISSUE TITLE
Module not found: Can't resolve '@microsoft/load-themed-styles' in '__directory\node_modules\@uifabric\styling\lib\styles'

### ISSUE THREAD

**Issue by yungfu** [OfficeDev/office-ui-fabric-react#3587]
### Bug Report

- __Package version(s)__: 5.34.0
- __Browser and OS versions__: win10, node_9.3.0, npm_5.5.1

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: (if applicable)

#### Describe the issue:

Failed to build with following error:
./node_modules/@uifabric/styling/lib/styles/theme.js
Module not found: Can't resolve '@microsoft/load-themed-styles' in '__directory\node_modules\@uifabric\styling\lib\styles'
Failed to compile.

This is the config.json:
{
  "name": "test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "office-ui-fabric-react": "^5.34.0",
    "react": "^16.2.0",
    "react-dom": "^16.2.0",
    "react-scripts": "1.0.17"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}

It seems a problem with my enviroment, but it worked well wihout the Office-UI-Fabric-React, and it worked well with the fabric in other machine. But I have cleaned all the cached npm global moduels, and downgrade the node to v8, the problem still exist in my enviroment.

**Comment by lambertwang** [comment]
What is your output of `rush install`?

**Comment by yungfu** [comment]
@lambertwang rush command not found!
This is a required dependency?

**Comment by lambertwang** [comment]
I thought this error was with building Fabric itself, but I was mistaken.  

Can you make sure that `@microsoft/load-themed-styles` is installed?

**Comment by yungfu** [comment]
@lambertwang , I can find this module in ./node_modules folder, and if I don't import any fabric component, it build successfully. I try to re-install the "@Microsoft/load-themed-styles", clean the npm cache, re-install all the modules, it still not work.
If I run the "npm list @microsoft/load-themed-styles", it will display "unment dependency" with serveral other modules.  I try to fix this problem from this issue, by "npm cache clear --force", remove "./node_modules", and remove all the global "./node_modules", it seems useless.

My environment install an old version nodejs and npm before, but I cannot remember which version it is. I didn't experience this kind problem on other modules.
My project worked well in the old version nodejs and npm before. I think this might not a "office-ui-fabric-react" issue, but I cannot identity the reason of the issue.

**Comment by yungfu** [comment]
@lambertwang finally, I fixed this issue. The problem was caused by the old npm, although "npm -v" command would return "5.5.0. I worked in windows, after I uninstall the nodejs, there is still some files remained in the "isntallpath/nodejs/node_modules/npm". After I removed all the residue files, and reinstall the nodejs v8, clear the npm cache, run "npm install" in project directory, everything go well.
I suspect the residue files may contains some cache global module dependent by the office-ui-fabric-react.
